.bazelrc

```diff
@@ -62,6 +62,14 @@ build:remote_cache --noslim_profile
 build:remote_cache --remote_cache=grpcs://remote.buildbuddy.io
 build:remote_cache --remote_timeout=3600
 
+build:sandboxed --strategy=SwiftCompile=sandboxed
```

Comment:
Is it possible to test sandboxed for all actions here and only allow certain ones to fallback to local instead? This might give us more info

Reply:
that's what we are doing here no? default setting has swiftCompile using worker and everything else local
This new config is everything sandboxed with a select set being allowed to fallback to local

Reply:
I meant, could we do `--strategy=sandboxed`

Reply:
hmm i guess you need it to override the above like:

```
build --strategy=SwiftCompile=worker
```

nvm this all looks good, thank you!